data/hai/hai reveal 4 middle.txt

```diff
@@ -76,15 +77,24 @@ mission "Hai Reveal [A02] Symbol Identification"
 			`	"Greetings Captain <last>, my name is Teeneep Goot'r. I'm a social advocate among the Hai, and I'm trying to help your friends with a problem. As you know, they were kidnapped by humans. A few other Hai and human victims are here; they were mostly rescued by other kind merchants in human space, but around half of those here are my staff."`
 			`	"My organization has been researching the disappearance of Hai ever since humans began disappearing here under similar circumstances. We've discovered some patterns to these abductions, and through those we've been able to reach some of the victims more quickly upon their return, and even rescue some ourselves. But we have noticed something strange among the abductees: those Hai that you rescued, and several others, too, are all experiencing mild cadmium poisoning."`
 			choice
-				`	"What patterns have you noticed in the disappearances?"`
-					goto patterns
-				`	"Tell me about the cadmium poisoning."`
-					goto cadmium
-				`	"How long have the disappearances been happening?"`
-					goto disappearances
+				`	"Before we discuss that, can you tell me if the Hai government is arming pirates?"`
+					goto direct
+				`	"It sounds like something I can help with, but there's something I need from you first."`
+					goto infotrade
+			label infotrade
+			`	She seems quite familiar with the practice of trading information, as she simply says, "What do you need?"`
+				goto tribute
+			label direct
+			`	Teeneep and the other Hai seem shocked. After a surprised exclamation in Hai, she says, "What!?"`
```

Comment:
Have we established a reason that you might be inclined to consider her your best avenue for asking up front?
Like she obviously is to us as the writers, but I think it feels perhaps a little rushed.
(If there's a lead-in justification somewhere else though feel free to point me at it, I may have overlooked it.)

Also, not sure if a surprised exclamation is appropriate here.
If you showed up and asked me something equivalently ridiculous (from her pov) like, "Is your boss selling hamsters to illegal pet traders?" my response would not be "What?!" and be more along the lines of "<concerned facial expression> ...are you high?"

Reply:
I have updated three missions, so the contact with Teeneep makes more sense. 

1. Sayari's note in the Mountaintop-furnishing mission vouches for Teeneep's organization and asks you to trade information and resources with her to solve the pirate troubles. (Her request is underscored by a substantial payment.) 
2. When you talk to Katya, she vouches for Sayari if you mention her by name, or is happy about a lead if you don't.
3. Before you ask Teeneep about the pirate arms deals, the narrator muses that this may be your only opportunity to pursue information, but warns you to be careful with your words. If you choose the direct approach, the Hai in the room laugh at you "for a while" until they see that you're serious.

>  If you showed up and asked me something equivalently ridiculous (from her pov) like, "Is your boss selling hamsters to illegal pet traders?" my response would not be "What?!" and be more along the lines of " ...are you high?"

Obviously, the problem is that you have a boring boss.